services/postgres/patched-entrypoint.sh

```diff
@@ -379,6 +383,11 @@ _main() {
 		else
 			echo "No post-init files found; ready for startup."
 		fi
+
+		# assumedly if we're here we've finished all init and post-init tasks
+		# so we create the /tmp/initialized file to signal that we're ready
+		# to the healthcheck script
```

Comment:
```suggestion
		# to run the healthcheck script
```

Reply:
The original wording is awkward, agreed, but the file actually does signal to the healthcheck script that the database is ready to accept connections. Perhaps it'd be better worded as "to signal to the healthcheck script that the database is ready to accept connections"? I'll revise it in any case, thanks.